src/main/java/seedu/foodrem/commons/enums/CommandType.java

```diff
@@ -16,209 +16,244 @@
  */
 public enum CommandType {
```

Comment:
Why was this file reorganised again? Was there any reason to do it? I reorganised it last time during refactoring because I thought organising them in alphabetical order would make it easier for developers to quickly jump to a section.

Reply:
I did not know it was in alphabetical order, I arranged it in terms of type of commands i.e. general commands, item commands and tag commands.

Reply:
I arranged it in the order the commands I felt will be used in.

Reply:
As in, I also arranged into the 3 groups
- General commands
- Item commands
- Tag command

But within each group, the commands are in alphabetical order cause I thought it would be easier that way,

Reply:
Ah I understand

Reply:
Will rearrange. Thanks for bringing this up